settings.php

```diff
@@ -975,38 +976,69 @@ function convertseconds($argument)
+                                                   be <code>192.168.47.0/24</code> and similar. If your network is larger, the CIDR has to be
+                                                   different, for instance a range of 10.8.0.1 - 10.8.255.255 results in <code>10.8.0.0/16</code>,
+                                                   whereas an even wider network of 10.0.0.1 - 10.255.255.255 results in <code>10.0.0.0/8</code>.
+                                                   Setting up IPv6 ranges is exactly similar to setting up IPv4 here and fully supported.
+                                                   Feel free to reach out to us on our
+                                                   <a href="https://discourse.pi-hole.net" target="_blank">Discourse forum</a>
+                                                   in case you need any assistance setting up local host name resolution for your particular system.</p>
+                                                <p>You can also specify a local domain name (like <code>fritz.box</code>) to ensure queries to
+                                                   devices ending in your local domain name will not leave your network, however, this is optional.
+                                                   The local domain name must match the domain name specified
+                                                   in your DHCP server for this to work. You can likely find it within the DHCP settings.</p>
+                                                <div class="form-group">
+                                                    <div>
+                                                        <input type="checkbox" name="rev_server" id="rev_server" value="rev_server" <?php if(isset($rev_server) && ($rev_server == true)){ ?>checked<?php } ?>>
+                                                        <label for="rev_server"><strong>Use Conditional Forwarding</strong></label>
+                                                    </div>
+                                                    <div class="input-group">
+                                                      <table class="table table-bordered">
+                                                        <tr>
+                                                          <th>Local network in <a href="https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing" target="_blank">CIDR notation</a></th>
+                                                          <th>IP address of your DHCP server (router)</th>
+                                                          <th>Local domain name (optional)</th>
+                                                        </tr>
+                                                        <tr>
+                                                          <div class="input-group">
```

Comment:
This is invalid BTW, you can't have a `div` inside a `tr`

Reply:
Thanks, addressed.